modules/otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -38,7 +33,15 @@ spec:
           receivers: [otlp]
           exporters: [prometheus]
 ----
-<1> Configures the Operator to create the Prometheus `ServiceMonitor` CR to scrape the Collector's internal metrics endpoint and Prometheus exporter metric endpoints. The metrics will be stored in the OpenShift monitoring stack.
+<1> Configures the {OTELOperator} to create the Prometheus `ServiceMonitor` CR or `PodMonitor` CR to scrape the Collector's internal metrics endpoint and the Prometheus exporter metrics endpoints.
```

Comment:
The pipe (|) in the example might be causing the colour formatting from exporters: onwards. I'm not going to hold this PR up as it could be out of scope. Maybe experimenting with escape characters might help: https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/#escape-with-backslashes or substitutions might help: https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/#the-subs-attribute

Reply:
We want to keep the pipe because it is even how the upstream community documents the objects.

In a following PR, I'll upgrade to the latest version of the CR which removes the pipes.

Reply:
I'd say that the pipe is doing to the block's formatting what it's supposed to do, IOW the coloring can be considered to be correct, so I wouldn't necessary try to find a solution to it.

(Alternatively, we could experiment with removing `yaml` from the block attributes, but the result might be even worse than what it is now.)